### ISSUE TITLE
Brew install doesnt work

### ISSUE THREAD

**Issue by KevinDoremy** [dral3x/StringsLint#31]
Error: 0.1.4 tag should be 05050f72afa49113bc6c8dacfd73e11a5bd88899
but is actually 54dd9876fc68a84668bd99640b61ca6ea7a15af8

**Comment by dral3x** [comment]
Hi @KevinDoremy 

I tested the brew install on my machine and it's working without any error.
Please do the following and let me know
```bash
brew uninstall stringslint
brew untap dral3x/dral3x
brew tap dral3x/dral3x
brew install stringslint
```
Thanks!

**Comment by KevinDoremy** [comment]
Hi @dral3x,

Thank you ! For your quick response.

It didn't work i'm still getting the same error after the brew install stringslint. 

Could it be link to this warning : "This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository."
 https://github.com/dral3x/StringsLint/commit/05050f72afa49113bc6c8dacfd73e11a5bd88899

**Comment by dral3x** [comment]
Thanks for checking @KevinDoremy!

I agree with you. That warning might be related to the problem you're facing.
Let me release a new version today, so I can unblock you.

**Comment by dral3x** [comment]
@KevinDoremy version 0.1.5 is out. Looks to work correctly on my machine. Please let me know
<img width="584" alt="Screenshot 2023-12-13 at 09 25 33" src="https://github.com/dral3x/StringsLint/assets/447976/4f245802-fbf7-4c66-9b81-3e080edd99bf">

**Comment by KevinDoremy** [comment]
==> Installing stringslint from dral3x/dral3x
==> swift build --disable-sandbox --configuration release --product stringslint
🍺  /opt/homebrew/Cellar/stringslint/0.1.5: 6 files, 1.6MB, built in 1 minute 47 seconds
==> Running `brew cleanup stringslint`..
This is working. Thank You !
